client/src/components/categorical/categorical.js

```diff
@@ -6,7 +6,7 @@ import * as globals from "../../globals";
 import Category from "./category";
 
 @connect(state => ({
-  categoricalSelectionState: state.controls.categoricalSelectionState
+  categoricalSelectionState: state.categoricalSelectionState
```

Comment:
A point for later, but `categoricalSelection` is probably the right name here, state is redundant.

Reply:
I'll fix, but for the record, I used the name that was previously in the code (ie, I didn't change it).

Reply:
Yup, this is my name

Reply:
fixed.